crd-generator/src/main/java/io/strimzi/crdgenerator/CrdGenerator.java

```diff
@@ -517,6 +518,13 @@ private ObjectNode buildBasicTypeSchema(AnnotatedElement element, Class type) {
         return result;
     }
 
+    private void addDescription(ObjectNode result, AnnotatedElement element) {
+        if (element.isAnnotationPresent(Description.class)) {
+            Description description = element.getAnnotation(Description.class);
+            result.put("description", description.value());
+        }
+    }
```

Comment:
Can you factor out and reuse the logic of `io.strimzi.crdgenerator.DocGenerator#getDescription`, which adds a full stop if one it missing? It's easier to do this than having to nit pick every time someone forgets to punctuate properly in `@Description`.